src/calcium/qqbar.jl

```diff
@@ -770,7 +782,7 @@ end
 Return the extension of the real sign function taking the value 1
 strictly in the right half plane, -1 strictly in the left half plane,
 and the sign of the imaginary part when on the imaginary axis.
-Equivalently, $\csgn(x) = x / \sqrt{x^2}$ except that the value is 0
+Equivalently, $\operatorname{csgn}(x) = x / \sqrt{x^2}$ except that the value is 0
```

Comment:
is \operatorname standard latex?

Reply:
Yes, it's the usual way to define a named operator.